tests/fetching/test_strat.py

```diff
@@ -0,0 +1,55 @@
+import pathlib
+import pytest
+from unittest import mock
+
+from fetching.projio import Project
+
+from fetching.strat import PyTestStrategy
+
+cwd = pathlib.Path.cwd() / "tests" / "resource" / "sample_decorator_appender_folder"
```

Comment:
Because it is in tests/fetching, it should be cwd = pathlib.Path.cwd().parent / "resource" / "sample_decorator_appender_folder"

Reply:
The cwd refers to the current working directory, i.e. the folder from which the program is executed FROM, not the program the file is in.

Reply:
But the test are run in that file. At least the path I have got from printing the cwd is tests/resource. Are you sure?

Reply:
The tests find the necessary files; I made sure of that by using `assert len(files) != 0` or something similar